testing/kfctl/kf_is_ready_test.py

```diff
@@ -145,6 +145,34 @@ def test_kf_is_ready(namespace, use_basic_auth, use_istio, app_path):
     logging.info("Verifying that deployment %s started...", deployment_name)
     util.wait_for_deployment(api_client, knative_namespace, deployment_name, 10)
 
+  if platform == "gcp":
```

Comment:
This should be a new test function in this module (a pytest module can have multiple test functions)

e.g.

```
test_workload_identity
```

We want separate functions to make it easier to distinguish failures with workload identity versus other tests.